pyodide_build/tests/recipe/_test_recipes/pkg_test_constraint/meta.yaml

```diff
@@ -0,0 +1,10 @@
+package:
+  name: pkg_test_constraint
+  version: "1.0.0"
+requirements:
+  constraint:
+    - numpy < 2.0
```

Comment:
Maybe numpy and scipy are bad examples though since realistically this is broken I'd assume?